Assets/General/Scripts/Console/ConsoleController.cs

```diff
@@ -55,6 +64,9 @@ private void Start()
         _playerTakeDamageButton.onClick.AddListener(HurtPlayer);
         _toggleGodMode.GetComponent<Button>().onClick.AddListener(ToggleGodMode);
         _playerInput.DebugConsole.OpenCloseConsole.performed += ctx => ToggleConsole();
+
+        //free look cam
+        _toggleFreeLookCamButton.GetComponent<Button>().onClick.AddListener(ToggleFreeLookCam);    
```

Comment:
If you're going to serialize these objects and they arent necessary for the game, please add a null check to prevent errors before `_freeLookCam` and `_toggleFreeLookCamButton`